apmpackage/apm/data_stream/internal_metrics_transaction/manifest.yml

```diff
@@ -0,0 +1,19 @@
+title: APM transaction metrics
+type: metrics
+dataset: apm.transaction
+dataset_is_prefix: true
+ilm_policy: metrics-apm.internal_metrics_transaction-default_policy
```

Comment:
Should we change the directory name to "transaction_metrics", to be more in line with the data stream name? Then the ILM policy name would be more consistent too.